Mixpanel/Mixpanel.m

```diff
@@ -191,6 +194,7 @@ - (instancetype)initWithToken:(NSString *)apiToken launchOptions:(NSDictionary *
 
         self.decideResponseCached = NO;
         self.showSurveyOnActive = YES;
+        self.enableABTestDesigner = YES;
```

Comment:
It is enabled by default on Android correct? So this would be parity with that SDK.